go/lint-project.sh

```diff
@@ -177,7 +159,7 @@ fi
 if [[ "$OS_NAME" != "windows" ]]; then
     wget -q -O - -q https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s v1.37.1
 
-    enabled="-E=bodyclose,exhaustive,rowserrcheck"
```

Comment:
Does `exhaustive` still run if we remove it from this `-E` flag?

Reply:
I thought we said since it's experimental to remove it

Reply:
Yes, but `exhaustive` was not being ran with the removal on Lint 180. I pushed a commit to enable it again.

Before:
```
INFO [lintersdb] Active 11 linters: [bodyclose deadcode gosimple govet ineffassign rowserrcheck staticcheck structcheck typecheck unused varcheck] 
```

After: 
```
INFO [lintersdb] Active 12 linters: [bodyclose deadcode exhaustive gosimple govet ineffassign rowserrcheck staticcheck structcheck typecheck unused varcheck] 
```

Reply:
Got it.  Thanks Adam!